pkg/edit/config_api.go

```diff
@@ -69,6 +69,44 @@ func callHooks(ev *eval.Evaler, name string, hook vals.List, args ...interface{}
 	}
 }
 
+func callFilters(ev *eval.Evaler, name string, filters vals.List, args ...interface{}) bool {
```

Comment:
Please add an elvdoc for `$edit:add-cmd-filters`, see https://github.com/elves/elvish/blob/master/CONTRIBUTING.md#documenting-builtin-functions-and-variables.